sycl/doc/extensions/experimental/sycl_ext_intel_fpga_task_sequence.asciidoc

```diff
@@ -0,0 +1,307 @@
+
+template <uint32_t Size>
+inline constexpr response_capacity::value_t<Size> response_capacity_v;
+
+} // namespace experimental
+} // namespace intel
+} // namespace ext
+} // namespace sycl
+----
+
+--
+[options="header"]
+|===
+| Property | Description
+| balanced | The `balanced` property is a guarantee to the SYCL device compiler that the `task_sequence` object will call exactly the same number of `async` s and `get` s over the object's lifetime (i.e. before the `task_sequence` destructor is invoked). 
+
+| invocation_capacity | The `invocation_capacity` property defines the minimum required number `Size` of `async` invocations that must be invokable without any `get` being called on that same set.  This number of `async` invocations without a `get` call is the minimum number that will be supported before the `async` member function blocks.
+
+| response_capacity | The `response_capacity` property tells the compiler the required simultaneous capacity `Size` of get calls. The compiler must guarantee storage capacity for results of up to `Size` `async` calls.
+|===
+--
+
+=== Compatibility with FPGA Kernel Interface Properties
+
+The Callable `f` defining the asynchronous task associated with a `task_sequence` may be declared with the FPGA Kernel Interface `pipelined` property described in link:../proposed/sycl_ext_oneapi_kernel_properties.asciidoc[sycl_ext_oneapi_kernel_properties].
+
+
+== Forward Progress Guarantees and Execution Model
+
+C{plus}{plus} defines a framework for describing the http://eel.is/c++draft/intro.progress[forward progress] of individual threads with respect to one another in a multi-threaded program. 
+
+Applying this framework to `task_sequence`, `async` tasks belonging to the same `task_sequence` object provide a http://eel.is/c++draft/intro.progress#11[weakly parallel forward progress guarantee]. That is, tasks belonging to a `task_sequence` object are not guaranteed to make forward progress with respect to each other. 
+
+Tasks belonging to different `task_sequence` objects provide a http://eel.is/c++draft/intro.progress#9[parallel forward progress guarantee] with respect to each other. That is, tasks belonging to different `task_sequence` objects are guaranteed to make forward progress with respect to each other once they have been initiated.
+
+Tasks belonging to a `task_sequence` object provide a http://eel.is/c++draft/intro.progress#9[parallel forward progress guarantee] with respect to their caller. That is, a `task_sequence`'s tasks are guaranteed to make forward progress with respect to their caller (that invoked `async`).
+
+When invoked through any call(s) to `async`, callable `f` behaves as if it is a SYCL kernel invoked via `single_task` in execution and memory model terminology.  One consequence of this is that a global memory synchronization point exists on invocation completion equivalently to kernel completion.
+
+== `task_sequence` Example Implementation 
```

Comment:
While useful for the implementation teams, I think we can remove this from the specification.

Reply:
Removed.